code/modules/mob/living/carbon/brain/brain_item.dm

```diff
@@ -59,7 +59,6 @@
 /obj/item/organ/brain/removed(var/mob/living/user)
 
 	if(!owner) return ..() // Probably a redundant removal; just bail
-	name = "[owner.real_name]'s brain"

```

Comment:
Nah, the brain renaming is handled in transfer_identity, this just caused the removal of a brain that was grafted onto an IPC to be renamed to the IPC's name, even if it was somebody else's